src/assets/languages/english.json

```diff
@@ -119,6 +119,11 @@
       "dashboardLink": "Go to Dashboard",
       "badgeLabel": "Expert"
     },
+    "disclaimer": {
+      "heading": "DISCLAIMER",
+      "body": "It has been tried best to ensure that the application is error free. If in any case, some error is found, PC-Prep-Kit shall not be made responsible. Please report such errors via GitHub issues or slack channel. ",
```

Comment:
Change the body to the following:

"In accord with our non-profit mission, the information - text, images and everything else present on this site has been researched and gathered from various legitimate sources and is being used purely for informational purposes only. 

The objective of this site is to promote malaria awareness through some exciting educational activities and games and was not intended to serve as a replacement for professional medical advice, diagnosis or treatment. Always seek the advice, diagnosis and/or treatment from qualified doctors for any malaria related help. 

If the imformation provided on this site is incorrect, misleading, invalid or outdated, please notify us on our Github repository <insert slack url as hyperlink for 'Github repository'> or on our slack channel <insert slack url as hyperlink for 'slack channel'>."

@lunayach does this sound good. Please suggest any changes that are necessary.

@Buddhiprabha @shreyans29 @paavininanda does this look fine? Please feel free to suggest any changes.

Reply:
Yes this looks good @Raxerz

Reply:
This is very good, please do the changes @lunayach